ls1intum/Artemis
# `Development`: Convert to new control flow syntax in Angular 17

### Checklist
#### General


#### Client

### Motivation and Context
The new control flow syntax is easier to understand.
Closes #7773.

### Description
The code is automatically converted using `ng generate @angular/core:control-flow`


### Testserver States
> [!NOTE]
> These badges show the state of the test servers.
> Green = Currently available, Red = Currently locked

![](https://byob.yarr.is/ls1intum/Artemis/artemis-test1)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test2)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test3)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test4)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test5)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test6)
![](https://byob.yarr.is/ls1intum/Artemis/artemis-test9)

### Testing steps

Test the main functionality and check that everything still works correctly.

Especially test places where we use multiple containers on the same page (e.g. the code editor, the assessment view of any exercise type, the exam mode) and check that the projected content is visible.

### Review Progress






#### Code Review
